src/controllers/update/overseas.entity.change.controller.ts

```diff
@@ -49,3 +52,40 @@ export const post = async (req: Request, resp: Response, next: NextFunction) =>
     next(errors);
   }
 };
+
+export const resetChangeData = async (req: Request, appData: ApplicationData) => {
+  if (appData && appData.update){
+    appData.who_is_registering = undefined;
+    appData.due_diligence = undefined
+    appData.overseas_entity_due_diligence = undefined;
+    appData.beneficial_owners_individual = undefined;
+    appData.beneficial_owners_corporate = undefined;
+    appData.beneficial_owners_government_or_public_authority = undefined;
+    appData.managing_officers_corporate = undefined;
+    appData.managing_officers_individual = undefined;
+    appData.beneficial_owners_statement = undefined;
+    appData.payment = undefined;
+    appData.trusts = undefined;
+    appData.update.registrable_beneficial_owner = undefined;
+    appData.update.bo_mo_data_fetched = false;
+    await existingBoMoForNoChange(req, appData);
+  }
+  return appData;
+};
+
+export const resetNoChangeData = (appData: ApplicationData) => {
+  if (appData.update){
```

Comment:
Again, I'm not sure update will be defined always. Maybe take all but appData.update out of if statement?